templates/jenkins-run.erb

```diff
@@ -75,4 +75,4 @@ if [ "$JENKINS_ENABLE_ACCESS_LOG" = "yes" ]; then
   PARAMS+=("--simpleAccessLogger.file=/var/log/jenkins/access_log")
 fi
 
-$JAVA_CMD "${PARAMS[@]}"
```

Comment:
It is my understanding that this way of using bash arrays actually should have the proper quotes.

Reply:
it's not about quotes in PARAMS array, it's about quotes in JENKINS_JAVA_CMD , JENKINS_JAVA_OPTIONS, JENKINS_HOME or JENKINS_WAR.